README.rst

```diff
@@ -1,23 +1,61 @@
-analyzere-python-viz |travis|
-=============================
 
-An extension to the analyzere python library that faciliates visualization of Analyze Re LayerView objects.
+Analyzere Python Client Extras  |travis|
+========================================
+
+
+An extension to the analyzere python library that facilitates "extras"
+including visualizations of Analyze Re LayerView objects.
 
 .. |travis| image:: https://travis-ci.org/analyzere/analyzere-python-extras.svg?branch=master
+    :alt: Travis-CI Build Status
     :target: https://travis-ci.org/analyzere/analyzere-python-extras
-    
+
 Installation
 ------------
 
 ::
 
-   pip install analyzere-viz
+   pip install analyzere_extras
 
 Usage
 -----
 
-Please see http://docs.analyzere.net/?python for the most up-to-date
-documentation.
+In order to make use of the visualization tools in the `analyzere_extras`
+module you will need to import the `analyzere` module.
+
+First you will need to define your connection information::
+
+   import analyzere
+   analyzere.base_url = '<your server url>'
+   analyzere.username = '<your userid>'
+   analyzere.password = '<your password>'
+
+Then you will need to query a LayerView that you would like to graph::
+
+   from analyzere import LayerView
+
+   lv = analyzere.LayerView.retrieve('011785b1-203b-696e-424e-7da9b0ec779a')
```

Comment:
I don't think it works on github (for whatever reason) but the [code and codeblocks directives](https://github.com/github/markup/issues/106) allow you to specificy a language.

If I remember correctly, this worked well in sphinx, but not on github.